ci/actions-templates/linux-builds-template.yaml

```diff
@@ -152,6 +163,18 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         if: github.event_name == 'push' && github.ref == 'refs/heads/stable'
+      - name: Build a snapcraft configuration file
+        run: |
+          sh ci/snapcraft.sh
+        if: matrix.snap_arch != ''
+        env:
+          DO_SNAP: 1
+          SNAP_ARCH: ${{ matrix.snap_arch }}
+          SNAP_EDGE: 1 # skip-stable
+      - uses: jhenstridge/snapcraft-build-action@master
```

Comment:
I've published a `v1` tag for this action.  I'd suggest using that instead of `master`.

Reply:
Yeah, this is actually a bogon from when I was diagnosing something in a fork of my own, thanks for the spot.